lib/chef/knife/ssh.rb

```diff
@@ -59,6 +59,12 @@ class Ssh < Knife
         :description => "QUERY is a space separated list of servers",
         :default => false
 
+      option :prefix,
+        :short => "-b ATTR",
+        :long => "--prefix-attribute ATTR",
+        :description => "The attribute to use for prefixing the ouput - default depends on the context",
+        :proc => Proc.new { |key| Chef::Config[:knife][:prefix_attribute] = key.strip }
```

Comment:
I'm reasonably certain that if you name your option `:prefix_attribute` and delete this proc that it merges correctly now (and has for a very long time, but we have never cleaned up this from the knife attributes), and in particular you should make sure that the Chef::Config option and the Mixlib::Cli option here match.